<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>System Wnioskowania Optymalnego Języka Programowania </string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>686</width>
      <height>432</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::WrapAllRows</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Czy zależy Ci na używaniu najnowocześniejszych rozwiązań?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>43</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QRadioButton" name="modernButton_0">
         <property name="text">
          <string>Nie</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="modernButton_1">
         <property name="text">
          <string>Raczej nie</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="modernButton_2">
         <property name="text">
          <string>Raczej tak</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="modernButton_3">
         <property name="text">
          <string>Tak, to dla mnie ważne</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Czy istotna jest dla Ciebie wydajność i optymalizacja projektu?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>43</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QRadioButton" name="perfButton_0">
         <property name="text">
          <string>Nie</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="perfButton_1">
         <property name="text">
          <string>Raczej nie</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="perfButton_2">
         <property name="text">
          <string>Raczej tak</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="perfButton_3">
         <property name="text">
          <string>Tak, to dla mnie ważne</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wybierz szacowaną złożoność projektu:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>34</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QRadioButton" name="compexityButton_0">
         <property name="text">
          <string>Mały</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_3</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="compexityButton_1">
         <property name="text">
          <string>Raczej mały</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_3</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="compexityButton_2">
         <property name="text">
          <string>Raczej duży</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_3</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="compexityButton_3">
         <property name="text">
          <string>Duży</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_3</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="compexityButton_4">
         <property name="text">
          <string>Ciężko określić</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_3</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Czy projekt musi być skalowalny?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>42</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QRadioButton" name="scalabilityButton_0">
         <property name="text">
          <string>Nie</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_4</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="scalabilityButton_1">
         <property name="text">
          <string>Raczej nie</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_4</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="scalabilityButton_2">
         <property name="text">
          <string>Raczej tak</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_4</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="scalabilityButton_3">
         <property name="text">
          <string>Tak, to dla mnie ważne</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_4</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Czy zależy Ci na popularności języka programowania?</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QRadioButton" name="popularityButton_0">
         <property name="text">
          <string>Nie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="popularityButton_1">
         <property name="text">
          <string>Trochę tak</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="popularityButton_2">
         <property name="text">
          <string>Tak, to dla mnie wazne</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Czy posiadasz doświadczenie programistyczne?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QRadioButton" name="expButton_0">
         <property name="text">
          <string>Tak</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_6</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="expButton_1">
         <property name="text">
          <string>Nie</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_6</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wybierz preferowany typ języka:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QRadioButton" name="typeButton_1">
         <property name="text">
          <string>Skryptowy</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_7</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="typeButton_2">
         <property name="text">
          <string>Kompilowany</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_7</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="typeButton_0">
         <property name="text">
          <string>Interpretowany</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_7</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="typeButton_3">
         <property name="text">
          <string>Obojętnie</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_7</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>821</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>831</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="Polish" country="Poland"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>System Wnioskowania Optymalnego Języka Programowania </string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>30</y>
      <width>821</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="locale">
     <locale language="Polish" country="Poland"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Odpowiedz na poniższe pytania</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="acceptButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kontynuuj</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_6"/>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup_5"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_7"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
 </buttongroups>
</ui>
